css_tasks/task9/scss/_pricing.scss

```diff
@@ -0,0 +1,120 @@
+@import "variables";
+@import "mixins";
+
+.pricing {
+  background: $mainBackgroundColor;
+  &__container {
+    @include flex;
+    justify-content: space-between;
+    margin: 0 auto;
+    padding: 5rem 0;
+  }
+  &__element {
+    @include flexColumn;
+    align-items: center;
+    width: 30%;
+    margin: 0 auto;
+    .pricing-element__type,
+    .pricing-element__price {
+      width: 100%;
+      color: $sectionColor;
+      background: $secondaryBackgroundColor;
+    }
+    .pricing-element__type {
+      padding: 1.25rem 0;
+      font-family: $primaryFont;
+      font-size: $sectionTitleFontSize;
+      font-weight: bold;
+      line-height: 1.2;
+      border: 1px solid $sectionColor;
+      text-align: center;
+    }
+    .pricing-element__price {
+      padding: 1.75rem 0 0;
+      font-family: $secondaryFont;
+      font-weight: bold;
+      color: $sectionColor;
+      text-align: center;
+      h3 {
+        padding: 1.75rem 0 0;
+      }
+      h3 span:first-child {
+        font-size: $jumbotronSubTitleFontSize;
+        line-height: 24px;
+      }
+      h3 span:last-child {
+        font-size: $priceFont;
+        line-height: 24px;
+      }
+      h4 {
+        padding: 1rem 0;
+        font-size: $navFontSize;
+        font-family: $primaryFont;
+        font-weight: 600;
+        line-height: 1.71;
+        color: $navbarColor;
+      }
+    }
+    .pricing-element__info {
+      width: 100%;
+      height: 5px;
+      background: $mainThemeColor;
+      text-align: center;
+      &--active {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        padding: 1.25rem 0;
+        font-family: $primaryFont;
+        font-size: $navFontSize;
+        font-weight: bold;
+        color: $sectionColor;
+        line-height: 1.71;
+        text-transform: uppercase;
+      }
+    }
+
+    .pricing-element__item {
+      padding: 0.75rem 0;
+      width: 100%;
+      font-family: $primaryFont;
+      font-size: $navFontSize;
+      color: $mainTextColor;
+      background: $sectionColor;
+      text-transform: uppercase;
+      text-align: center;
+      box-shadow: inset 0 -1px 0 0 $mainBackgroundColor;
+    }
+    &--active {
```

Comment:
make container flex and all elements to bottom, it would be better, plus modifiers should be close to class which they modify, not so far that if it hard to find one

and generally it is not an active pricing, it is more which marketing what you to buy at first place, or average user segmentation/most popular/  :D

Reply:
Which container? .pricing__element? It's already flex. What is wrong with this approach?

Reply:
to .pricing__container just add align-items: flex-end, and there would be no reason for this class at all

nothing wrong with this approach, there is just better one